src/md/src/TipsAndTricks/index.md

```diff
@@ -1,15 +1,15 @@
 # Tips and tricks
 
->If you see inconsistencies, typos or want to add something, please, [send us Pull Request into source](https://github.com/reportportal/documentation/edit/master/src/md/src/TipsAndTricks/index.md)
+>If you see inconsistencies, typos or want to add something, please, [send us Pull Requests to branch it into our source code](https://github.com/reportportal/documentation/edit/master/src/md/src/TipsAndTricks/index.md)
```

Comment:
some most natural for open source and github users phrase would be just 'send us pull request'
Pull request is Github/git term, i would suggest to not change it